pkg/db/db.go

```diff
@@ -20,14 +20,13 @@ var DB *gorm.DB
 
 // GetUrl Get database config and return url
 func GetUrl() string {
-	conf := config.Get()
 	return fmt.Sprintf(
 		"user=%s password=%s dbname=%s host=%s port=%d sslmode=disable",
-		conf.DBUser,
-		conf.DBPassword,
-		conf.DBName,
-		conf.DBHost,
-		conf.DBPort,
+		config.Config().Database.User,
```

Comment:
Should we be calling the config function every time we need this? I think the previous method of assigning a variable to conf (config.Config().Database) would be more ideal no?

Reply:
In my (javascript) mind, this is running the same function 5 times, when it only needs to be instantiated once.  Is there Golang magic that I am unaware of?

Reply:
Its largely for ease of use.  config.Config() lazily-loads the config and returns the struct.  If we refer to the struct directly, we have to make sure to call a Load() method prior to anytime that is needed, which means in every single test (or really every MainTest of every package).  It seemed 'nicer' to just call this to have it lazy-loaded, and then we never have to explicitly call Load()  anywhere (every command, every test, etc..)

Reply:
although i should have it return a pointer, and not the actual struct!   i'll make that change.  Let me know if you prefer us calling Load() directly everywhere and I can think about it a bit more

Reply:
I don't think we need Load() everywhere, if this is the drawback between the two ideas.

Reply:
You have to have Load() called before any reference to config values, so you have to have Load() called early in the process everywhere (tests, commands, etc..)

Reply:
Although we have to do that with the database today.  So maybe having something like  `config.Bootstrap()` isn't a terrible idea?  It could handle both

Reply:
Does Load just have to be called once early in the chain? Like during an Init() function for example? or is it  everytime 
we access a value?